lib/bootloader_setup.pm

```diff
@@ -189,6 +189,7 @@ sub boot_local_disk {
 sub boot_into_snapshot {
     send_key_until_needlematch('boot-menu-snapshot', 'down', 10, 5);
     send_key 'ret';
+    record_soft_failure 'bsc#1017558:Cannot view timestamp of read-only snapshots in GRUB as names truncated' if is_leap;
```

Comment:
We should only record a soft failure when we detected an error condition. Also, I do not think that openSUSE Tumbleweed is different in general only that more recent Tumbleweed seems to not show the selection list when there is just one snapshot. But I assume that would apply to openSUSE Leap 15.1 as well.

On top I am not sure if the format "bsc#[0-9]:…" can not be correctly parsed by openQA which you can also see in the verification run.

So in light of this, sorry, but the PR should not have been merged at all and needs to be corrected.

Reply:
https://openqa.opensuse.org/tests/717024#step/grub_test/6 works fine

Reply:
Correct, looks fine. Please don't overlook my other two remarks